tests/unit/components/playback/controls/RecordingTime.spec.js

```diff
@@ -78,16 +78,22 @@ describe("RecordingTime.vue", () => {
     await store.dispatch("playback/start_recording");
     await wrapper.vm.$nextTick(); // wait for update
 
-    expect(wrapper.find(".span__recording-text").text()).toEqual("Recording:");
-    expect(wrapper.find(".span__time-text").text()).toEqual("00:00:00.000");
+    expect(wrapper.find(".span__recording-text").text()).toStrictEqual(
+      "Recording:"
+    );
+    expect(wrapper.find(".span__time-text").text()).toStrictEqual(
+      "00:00:00.000"
+    );
     await wrapper.vm.$nextTick(); // wait for update
 
     store.commit("playback/set_x_time_index", 6736300);
     await wrapper.vm.$nextTick(); // wait for update
 
-    expect(wrapper.find(".span__time-text").text()).toEqual("00:00:55.018");
+    expect(wrapper.find(".span__time-text").text()).toStrictEqual(
+      "00:00:55.018"
+    );
   });
-  test("When Recording Start is initiated moved by 55018 intermediate x_time_index and Recording Stop", async () => {
+  test("Given that RecordingTime is mounted successfully from the build dist file, When Recording Stop is initiated at x_time_index moved by 55018, Then the text updates and the text is null", async () => {
```

Comment:
the component is mounted from the .vue file, not the dist file.  Also, it looks like there are other conditions for setting up this test--it looks like "start_recording" is initiated and then Stop Recording is triggered